.golangci.json

```diff
@@ -3,11 +3,28 @@
 		"disable-all": true,
 		"enable": [
 			"govet",
-			"golint",
-			"goimports"
+			"revive",
+			"goimports",
+      "gofmt"
 		]
 	},
 	"run": {
 		"deadline": "5m"
-	}
+	},
+  "issues": {
+		"exclude-rules": [
+			{
+				"linters": "typecheck",
+				"text": "imported but not used"
+			},
+			{
+				"linters": "revive",
+				"text": "should have comment"
+			},
+			{
+				"linters": "revive",
```

Comment:
Why are we excluding this check?

Reply:
I copied the exclude-rules from the k3s config. Let's see what happens if I don't exclude it.